modules/effects/src/effects_resolver.ts

```diff
@@ -61,3 +50,19 @@ export function mergeEffects(
 
   return merge(...observables$);
 }
+
+/**
+ * @internal
+ */
+export function resubscribeInCaseOfError<T extends Action = Action>(
```

Comment:
Is `= Action` needed? I think just extending Action is fine.

Reply:
No, it isn't needed. Thanks for the review 👍